app/webpack/users/edit/components/relationships.jsx

```diff
@@ -0,0 +1,205 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Pagination from "rc-pagination";
+import moment from "moment";
+
+import SettingsItem from "./settings_item";
+import RelationshipsCheckboxContainer from "../containers/relationships_checkbox_container";
+import UserFollowing from "./user_following";
+import BlockedMutedUsersContainer from "../containers/blocked_muted_users_container";
+
+const Relationships = ( {
+  relationships,
+  filteredRelationships,
+  updateFilters,
+  sortRelationships,
+  showModal,
+  loadPage,
+  page
+} ) => {
+  const showRelationships = ( ) => filteredRelationships.slice(
+    page * 10 - 10, page * 10 // only display up to 10 items on current page
+  ).map( user => {
+    const { friendUser } = user;
+
+    return (
+      <div className="row relationship-row-margin" key={friendUser.login}>
+        <div className="divider relationship-row-margin" />
+        <div className="col-xs-4">
+          <UserFollowing user={friendUser} />
+        </div>
+        <div className="col-sm-8">
+          <div className="row">
+            <div className="col-md-6">
+              <RelationshipsCheckboxContainer
+                name="following"
+                label={I18n.t( "following" )}
+                id={user.id}
+              />
+              <RelationshipsCheckboxContainer
+                name="trust"
+                label={I18n.t( "trust_with_private_coordinates" )}
+                id={user.id}
+              />
+              {user.reciprocal_trust && <dfn>{I18n.t( "user_trusts_you_with_their_private_coordinates", { user: friendUser.login } )}</dfn>}
+            </div>
+            <div className="col-md-6">
+              <dfn className="relationship-row-margin">
+                {`${I18n.t( "added" )} ${moment( friendUser.created_at ).format( "LL" )}`}
+              </dfn>
```

Comment:
Have to admit I need to look up the `<dfn>` tag. How is the date added a definition?

Also, it would be better to use the `added_on_datetime` translation key or to add a new one like `added_datetime` so a) translators can translated added" in a way appropriate to the context, and b) translators can control word order.

Reply:
Switched to `<em>` and the full sentence `added_on_datetime` translation key.

Reply:
`<em>` is fine, but fwiw, if you need an element and there isn't a good semantic match, `<div>` and `<span>` are usually what I use. A reader (machine or otherwise) assuming the tags were being used with their implied meanings might interpret this as a particularly emphatic date, when it's really not more important than the other text that surrounds it. `<em>` also comes with some default visual styling (italic in most browsers) that you may not want and/or may have to override in CSS.